client/disputes/index.tsx

```diff
@@ -143,6 +144,46 @@ const getHeaders = ( sortColumn?: string ): DisputesTableHeader[] => [
 	},
 ];
 
+/**
+ * Returns a smart date if dispute's due date is withint 72 hours.
+ * Otherwise, returns a date string.
+ *
+ * @param {CachedDispute} dispute The dispute to check.
+ *
+ * @return {ReactNode | string} If dispute is due within 72 hours, return the element that display smart date. Otherwise, a date string.
```

Comment:
Is `ReactNode` the correct return value type?

Reply:
I think either `ReactNode` (which includes `string`)  `string | JSX.Element` is OK. I'd lean towards `string | JSX.Element` because it's more explicitly letting us know what it returns.

TS inferred types ignore the JS docblock return type anyway.

<img width="621" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/3ce07a7a-aae2-49b3-aaf9-3a1c8f59c402">

Reply:
I've updated this in 89f59b2